app/controllers/experiments_controller.rb

```diff
@@ -369,11 +371,53 @@ def create
         replication_level: [:optional, :security_default, :integer, :positive],
         execution_time_constraint: [:optional, :security_default, :integer, :positive],
         parameter_constraints: [:optional, :security_json],
-        type: [:optional, :security_default]
+        type: [:optional, :security_default],
+        workers_scaling: [:optional] # TODO boolean validator
```

Comment:
To leci z JS. Jeśli użyjemy np. curla to rozumiem, że query string ma wyglądać tak: ``?workers_scaling=true`` ?
To w takim razie, żeby to TODO nie umknęło do wdrożenia, bo wtedy np. False będzie znaczyć true.